src/dataflow/components/dataflow-program.tsx

```diff
@@ -484,6 +484,30 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
     return true;
   }
 
+  private checkActiveRelaysAndRunProgram = () => {
+    const { ui } = this.stores;
+    fetchActiveRelays().then((result: any) => {
```

Comment:
A quick and rough scan seems to show that this project uses `async/await` more than `then`, including in this file, and I think it's a nicer syntax in general, but that's a personal style question.